libvirt.go

```diff
@@ -300,6 +302,52 @@ func (d *Driver) Create() error {
 	return d.Start()
 }
 
+func (d *Driver) createDisk() error {
+	u, err := url.Parse(d.DiskPathURL)
+	if err != nil {
+		return fmt.Errorf("Invalid URL: %s", d.DiskPathURL)
+	}
+
+	if u.Scheme == "file" || u.Scheme == "" {
+		// For local image on Linux, we can avoid copying by making use of base image through qemu-img
+		// Local disk image so we can avoid copying by making use of backing image feature of qcow2
+		diskPath := u.Path
+		machineIsoPath := filepath.Join(d.StorePath, "machines", d.MachineName, "crc")
+
+		log.Infof("Create disk image at %s (backing store: %s)", machineIsoPath, diskPath)
+		command := exec.Command("qemu-img", "create", "-b", diskPath, "-f", "qcow2", machineIsoPath)
+		output, err := command.CombinedOutput()
+		log.Debugf("Output of `%s` command:\n%s", commandAsString(command), output)
+		if err != nil {
+			log.Warnf("Error occurred running `%s` command: %s", commandAsString(command), err)
+		} else {
+			return nil
+		}
```

Comment:
@zeenix (minor) This can be done as follow, I don't think you need an if/else here.
```golang
if err == nil {
     return nil
}
log.Warnf("Error occurred running `%s` command: %s", commandAsString(command), err)
```